docs/corpora_schema.md

```diff
@@ -119,15 +136,12 @@ layer\_descriptions|Mapping from {layer\_name: layer\_description, ...} Each des
 
 There are also fields that are required so that the cellxgene Data Portal and Explorer can present datasets appropriately.
 
-Each dataset must have at least one **embedding**, a mapping from each cell to a tuple of floats of length at least 2. These are usually generated by algorithms
-like umap or tsne and are used to display the dataset in the Explorer. cellxgene provides a command line tool to add embeddings if they are missing.
-
-Datasets also must have a few other metadata fields for presentation:
-
-**Field name**|**Description**
-:--|:--
-title|String that identifies the dataset
-
+* Each dataset MUST have at least one **embedding**, a mapping from each cell to a tuple of floats of length at least 2. These are usually generated by algorithms
+  like umap or tsne, but can also coordinates of cells in spatial assays. They are used to display the dataset in the Explorer.
```

Comment:
_can also coordinates_ ?

Reply:
Updated to state _"but can also represent `(x, y)` coordinates of cells in spatial assays"_. Thanks.